lib/rules/no-throw-literal.js

```diff
@@ -1,12 +1,32 @@
 /**
  * @fileoverview Rule to restrict what can be thrown as an exception.
  * @author Dieter Oberkofler
+ * @copyright 2015 Ian VanSchooten. All rights reserved.
  * @copyright 2015 Dieter Oberkofler. All rights reserved.
  */
 
 "use strict";
 
 //------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Determine if a node is considered literal (including concatenation)
+ * @param  {ASTNode}  node  ASTNode to check for only literals
+ * @returns {Boolean}       True if considered literal, false otherwise
+ */
+function isLiteral(node) {
+    if (node.type === "Literal") {
+        return true;
+    }
+
+    if (node.type === "BinaryExpression") {
+        return true;
```

Comment:
He mentioned it on the issue as well. Okay, let's go with his approach...I'll deal with my pedantry regarding the rule name in private. :)

I would suggest, though, that the function be named `isErrorObject()`...the "not" confused me.